alphadia/workflow/peptidecentric.py

```diff
@@ -319,119 +314,85 @@ def get_batch_plan(self):
 
         return plan
 
-    def start_of_calibration(self):
-        self.batch_plan = self.get_batch_plan()
+    def extract_optimization_data(self, target, min_iterations):
```

Comment:
Good idea to put this into in a separate method.
Can we align the name more so it becomes clear what process this is?

Can youy check if we need to pass `target` and `min_iterations` as arguments?
Otherwise we could read them directly from the log.
(Nit) lets give them the same name as the parameters in the log.